.travis.yml

```diff
@@ -4,17 +4,34 @@ sudo: false
 cache:
   - pip
 
+before_cache:
+  - rm ~/.cache/pip/log/debug.log
+
 language: python
 python:
   - "2.7"
   - "3.5"
 
 install:
-  - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
-  # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
-  - ansible-playbook --syntax-check playbooks/byo/config.yml
   - tox
-  - cd utils && tox
+
+notifications:
+  email:
+    recipients:
+      - jdetiber@redhat.com
+      - sdodson@redhat.com
```

Comment:
This might be very noisy. I just want to make sure @sdodson is :+1: for that. I'd hate for the emails to be to much and simply be deleted.